core/remote/watcher.js

```diff
@@ -318,19 +321,23 @@ class RemoteWatcher {
             continue
           } else {
             _.assign(previousChange, {
-              type: 'IgnoredChange',
+              type: 'DescendantChange',
               detail: `Folder was moved as descendant of ${change.doc.path}`
             })
+            // $FlowFixMe
+            remoteChange.includeDescendant(change, previousChange)
             continue
           }
         } else if (remoteChange.isChildMove(previousChange, change)) {
-          return {
+          const descendantChange = {
             sideName,
-            type: 'IgnoredChange',
+            type: 'DescendantChange',
             doc,
             was,
             detail: `Folder was moved as descendant of ${_.get(previousChange, 'doc.path')}`
           }
+          previousChange.type === 'DirMove' && remoteChange.includeDescendant(previousChange, descendantChange)
```

Comment:
`if` block?

Reply:
I know you love one-liners, but `if` blocks are probably more common in the current codebase ;)